tryiiif/home/__init__.py

```diff
@@ -30,7 +30,13 @@ def index():
         manifest = make_manifest(uid, url, b64url, name, info['height'],
                                  info['width'])
         rc.conn.set(uid, json.dumps(manifest))
-        return redirect(url_for('viewers.viewer', viewer='uv', uid=uid))
+
```

Comment:
In order to avoid a proliferation of ifs, I'd recommend something like the following:

```python
return redirect(url_for('viewers.viewer', viewer=request.form['submit'], uid=uid))
```